single_tech_samples/datafactory/sample2_cicd_auto_publish/README.md

```diff
@@ -0,0 +1,322 @@
+# Automated Publish and CI/CD for Azure Data Factory
+
+This sample demonstrates the deployment of Azure Data Factory using Automated Publish Method. Usually, the Azure Data Factory deployment required Manual Publish setup where the developer had to manually Publish the ADF changes from the portal. This step generated the ARM Templates which were used in deployment step.
+
+In this sample, we have eliminated the Manual Publish step by using ADF Packages for Automated Publish. Now, the ARM templates will be generated automatically without any manual intervention.
```

Comment:
In this sample, the need to manually publish step has been eliminated

Reply:
ADF Packages for Automated Publish. => by using a publicly available npm package [@microsoft/azure-data-factory-utilities](https://www.npmjs.com/package/@microsoft/azure-data-factory-utilities) for automated publishing.

Reply:
Now, the ARM templates will be generated automatically without any manual intervention. => With this, the ARM templates can be generated automatically without any manual intervention.

Reply:
Done